src/openrct2/peep/Peep.h

```diff
@@ -60,6 +60,16 @@ enum class PeepType : uint8_t
     Invalid = 0xFF
 };
 
+enum class StaffType : uint8_t
```

Comment:
Move back to staff.h

Reply:
I guess he did it because he needed it for the member variable on peep